tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -5304,56 +5304,65 @@ inline void PadImageStyle(const tflite::PadParams& op_params,
 }
 
 template <typename T>
-inline void Slice(const tflite::SliceParams& op_params,
+inline void Slice(tflite::SliceParams& op_params,
```

Comment:
In Google coding style, we just use const reference or pointer, reference is not allowed. If you follow the following comment, I think const tflite::SliceParams& will be just fine.